converter/internal/prometheusconvert/scrape/scrape.go

```diff
@@ -0,0 +1,59 @@
+package scrape
+
+import (
+	"time"
+
+	"github.com/grafana/agent/component/discovery"
+	"github.com/grafana/agent/component/prometheus/scrape"
+	"github.com/grafana/agent/converter/internal/prometheusconvert/common"
+	promconfig "github.com/prometheus/prometheus/config"
+	promdiscovery "github.com/prometheus/prometheus/discovery"
+	"github.com/prometheus/prometheus/storage"
+)
+
+func Reconvert(scrapeConfig *promconfig.ScrapeConfig) *scrape.Arguments {
+	if scrapeConfig == nil {
+		return nil
+	}
+
+	return &scrape.Arguments{
+		Targets:               getTargets(scrapeConfig),
+		ForwardTo:             []storage.Appendable{}, // TODO
+		JobName:               scrapeConfig.JobName,
+		HonorLabels:           scrapeConfig.HonorLabels,
+		HonorTimestamps:       scrapeConfig.HonorTimestamps,
+		Params:                scrapeConfig.Params,
+		ScrapeInterval:        time.Duration(scrapeConfig.ScrapeInterval),
+		ScrapeTimeout:         time.Duration(scrapeConfig.ScrapeTimeout),
+		MetricsPath:           scrapeConfig.MetricsPath,
+		Scheme:                scrapeConfig.Scheme,
+		BodySizeLimit:         scrapeConfig.BodySizeLimit,
+		SampleLimit:           scrapeConfig.SampleLimit,
+		TargetLimit:           scrapeConfig.TargetLimit,
+		LabelLimit:            scrapeConfig.LabelLimit,
+		LabelNameLengthLimit:  scrapeConfig.LabelNameLengthLimit,
+		LabelValueLengthLimit: scrapeConfig.LabelValueLengthLimit,
+		HTTPClientConfig:      *common.ReconvertHttpClientConfig(&scrapeConfig.HTTPClientConfig),
+		ExtraMetrics:          false,
+		Clustering:            scrape.Clustering{Enabled: false},
+	}
+}
+
+func getTargets(scrapeConfig *promconfig.ScrapeConfig) []discovery.Target {
+	targets := []discovery.Target{}
+
+	for _, serviceDiscoveryConfig := range scrapeConfig.ServiceDiscoveryConfigs {
```

Comment:
Can you go through this entire PR and add TODO comments for anything from Prometheus which isn't being converted yet? 

Since there's a lot of surface area, I'm worried we'll forget something.